builder/vsphere/common/step_keep_cdrom.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+//go:generate packer-sdc struct-markdown
+//go:generate packer-sdc mapstructure-to-hcl2 -type KeepCDRomConfig
+
+package common
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+	"github.com/hashicorp/packer-plugin-vsphere/builder/vsphere/driver"
+)
+
+type KeepCDRomConfig struct {
+	// Keep a number of CD-ROM devices. Range: 1-4.
+	// If set to 0, `keep_cdrom` is ignored and the step is skipped.
+	// When set to a value in the range, `remove_cdrom` is ignored and
+	// the CD-ROM devices are kept without any attached media.
+	KeepCdrom int32 `mapstructure:"keep_cdrom"`
```

Comment:
Why not just make this an int? 
```suggestion
	KeepCdrom int `mapstructure:"keep_cdrom"`
```

Reply:
Resolved in 011dd5a.